### DISCUSSION TITLE
Source html edit possibility

### DISCUSSION THREAD

**Discussion by ylsv** [taiga-family/taiga-ui#10061] [General]
Is it possible to integrate source html edit option to this editor? Didn't find such option for now. Maybe you could advise some extensions / settings / ways to achieve this result?
What I need is a button let it be 'source'. I click on it and see the html with tags, that is editable. After I make changes to html and switch this button back to view mode it applies changes.
E.g. like made here: https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html
![image](https://github.com/user-attachments/assets/d82f6ced-d8b8-4472-bdf3-3e84a63f5fe2)  

![image](https://github.com/user-attachments/assets/3a5aebdd-053b-4e7d-8894-292a98ad093f)

**Comment by splincode** [comment]
Opened issue: https://github.com/taiga-family/editor/issues/1632

Right now we don't have any solution
